### ISSUE TITLE
LaTeX style file

### ISSUE THREAD

**Issue by sepideh-esk** [openjournals/joss#1161]
When I inspect the PDFs of JOSS I see that they are created by "LuaHBTeX, Version 1.15.0 (TeX Live 2022)".

Do have the (La)TeX style file for directly compiling the PDF from a LaTeX source using TeX Live 2022? This would help for arXiv submission also.

**Comment by jedbrown** [comment]
It's `pandoc` driving `lualatex`. You can see the invocation here. https://github.com/openjournals/inara/blob/main/Makefile#L28-L42
https://github.com/openjournals/inara/blob/main/data/defaults/pdf.yaml

If you make a target for the raw latex, we'd welcome a PR.

**Comment by xuanxu** [comment]
You can select preprint as output when running [Inara](https://github.com/openjournals/inara) using the flag `-o preprint` and it will generate a LaTeX file suitable to send to preprint archives.

**Comment by pedramardakani** [comment]
> It's `pandoc` driving `lualatex`. You can see the invocation here. https://github.com/openjournals/inara/blob/main/Makefile#L28-L42 https://github.com/openjournals/inara/blob/main/data/defaults/pdf.yaml
> 
> If you make a target for the raw latex, we'd welcome a PR.

I can make it if no one's started yet ✌

**Comment by pedramardakani** [comment]
> You can select preprint as output when running [Inara](https://github.com/openjournals/inara) using the flag `-o preprint` and it will generate a LaTeX file suitable to send to preprint archives.

Oh good call @xuanxu ! Here's what I did when running `inara` with the `docker` image:

```bash
sudo docker run --rm -it -v $PWD:/data \
            -u $(id -u):$(id -g) --entrypoint="" openjournals/inara \
            sh -c 'inara paper.md -o preprint && pdflatex paper.preprint.tex'
```

So the `entrypoint` is reset, and then, as @xuanxu said, the `preprint` option makes the `paper.preprint.tex` file. Right after that, we're using `pdflatex` to create the pdf _inside_ the docker environment.

I tried building it on the _host_ machine as well, but it complained about not finding `authblk.sty` and `orcidlink.sty` files. However, these can be installed pretty easily using: `tlmgr install preprint orcidlink`. So now I can get the preprint, tinker it and build it on the host machine quickly 🤓

I hope this helps @sepideh-esk 🤞
